generators/client/templates/_bower.json

```diff
@@ -4,46 +4,46 @@
   "appPath": "<%= MAIN_SRC_DIR %>",
   "testPath": "<%= TEST_SRC_DIR %>spec",
   "dependencies": {
-    "angular": "1.5.5",
-    "angular-aria": "1.5.5",
-    "angular-bootstrap": "1.3.3",
+    "angular": "1.5.8",
+    "angular-aria": "1.5.8",
+    "angular-bootstrap": "2.0.0",
```

Comment:
@deepu105 I've confirmed this upgrade works by running the Protractor tests locally (using `gulp itest`) and having them pass. I also started the app and ensured there were no console errors.